nixos/modules/services/hardware/monado.nix

```diff
@@ -0,0 +1,71 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkEnableOption mkPackageOption;
+
+  cfg = config.services.monado;
+in
+{
+  options.services.monado = {
+    enable = mkEnableOption "monado";
+
+    package = mkPackageOption pkgs "monado" { };
+  };
+
+  config = mkIf cfg.enable {
+    security.wrappers."monado-service" = {
+      owner = "root";
+      group = "root";
+      # cap_sys_nice needed for asynchronous reprojection
+      capabilities = "cap_sys_nice+eip";
+      source = "${cfg.package}/bin/monado-service";
+    };
+
+    systemd.user = {
+      services.monado = {
+        description = "Monado XR runtime service module";
+        requires = [ "monado.socket" ];
+        conflicts = [ "monado-dev.service" ];
+
+        unitConfig.ConditionUser = "!root";
+
+        environment = {
+          XRT_COMPOSITOR_LOG = "debug";
+          XRT_PRINT_OPTIONS = "on";
+          IPC_EXIT_ON_DISCONNECT = "off";
```

Comment:
These should probably be optional and off by default, not?

Reply:
These are the default values of the upstream unit configuration: https://gitlab.freedesktop.org/monado/monado/-/blob/f84353ded3c5ecb08b763a3e061488b2b73a6b3b/src/xrt/targets/service/monado.in.service